src/Analyzers/Analyzers.csproj

```diff
@@ -29,6 +30,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.4" PrivateAssets="all" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.9.2" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.ResxSourceGenerator" Version="3.11.0-beta1.24165.2" PrivateAssets="all" />
```

Comment:
Yes, there is no other stable package: https://www.nuget.org/packages/Microsoft.CodeAnalysis.ResxSourceGenerator.

Reply:
How come we don't need this package in the In proc analyzer?

Reply:
This is a new approach to avoid having generated code being committed to the repository. This was a (really really nice) suggestion/idea from Jacob in this very same PR discussion: https://github.com/microsoft/durabletask-dotnet/pull/284#discussion_r1560063036.

The reason why this wasn't needed it was because it used the classical approach of having the generated code by VS and committing it to git. (instead of generating it at compile time).

Reply:
I see, thanks for the explanation!